src/test/resources/omop/settings.json

```diff
@@ -170,37 +170,111 @@
       ]
     }
   ],
-  "featureValueGroups": [
+  "datasetConceptSets": [
     {
       "name": "Drug",
-      "values": []
+      "table": {
+        "datasetTableName": "drug_exposure",
+        "columns": [],
+        "primaryTableRelationship": "fpk_person_drug",
+        "secondaryTableRelationships": [
+          "fpk_drug_type_concept",
```

Comment:
I think that we should be leveraging TDR's relationship objects here instead of storing fpk names. (So instead of storing the primary table fpk, we'd store the field of the table that contains the value which has a fpk relationship to the primary table.)  But that change can be done later, since this PR is about changing where this data is stored, not how it's represented.